tests/php/test_fieldmanager_field.php

```diff
@@ -925,6 +925,29 @@ public function test_description_escaping() {
 		$this->assertContains( $description_html, $html );
 	}
 
+	public function test_button_content_escaping() {
+		$id                  = rand_str();
+		$button_content_raw  = rand_str();
```

Comment:
I'm assuming this was cribbed from another test, but we've since stopped using `rand_str()`. It's better to use static strings, since unit tests should always be as reliable and repeatable as possible.